.github/workflows/main.yml

```
@@ -0,0 +1,19 @@
+      - main
+      - deploy/aws-s3-ec2 
+    paths-ignore:
+      - README.md
+
+jobs: 
+  integration:
+    name: Continuous Integration
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)